third-party/realdds/include/realdds/dds-device.h

```diff
@@ -25,7 +25,8 @@ class dds_participant;
 // Represents a device via the DDS system. Such a device exists as of its identification by the device-watcher, and
 // always contains a device-info and GUID of the remote DataWriter to which it belongs.
 // 
-// The device may not be ready for use (will not contain sensors, profiles, etc.) until it is "run".
+// The device may not be ready for use (will not contain sensors, profiles, etc.) until it received all handshake
+// notifications from the server, but it will start receiving notifications and be able to send controls right away.
```

Comment:
Why would we want to handle notifications or send controls before we are ready?

Reply:
You might just be listening to a device's notifications, and do not care about streaming or streams.

Reply:
For controls, it's a good question which I've asked myself. But it's a similar situation, I think: it's up to the server to say it's not ready, not the client, IMO. I.e., if a control can be sent, why disable it? Worst case, you get an error back because of a timeout in the reply.

Reply:
(also note that, for testing, that's exactly what I did: a server that does no handshake -- therefore the client device will never be ready)

Reply:
OK